scripts/zones/Yuhtunga_Jungle/mobs/Carthi.lua

```diff
@@ -15,18 +15,18 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
+    if (isKiller) then -- Causes this to only run once per Carthi kill
```

Comment:
isKiller returns true or false on the player that landed the killing blow. I'm 99% sure it will return false if it was killed by DoT while unclaimed.